src/rgblob/game.cljs

```diff
@@ -0,0 +1,218 @@
+(ns rgblob.game
+    (:require [clojure.string :as string]
+              [rgblob.sprite :as rs]
+              [rgblob.player :as player]
+              [rgblob.util :as util :refer [log-obj log]]))
+
+(defonce ^:private state (atom {:render-list []}))
+
+(defn get-player []
+    (:player @state))
+
+(def board [[:white :white :blue :white :white :white :green :green]
+            [:white :green :green :green :red :white :black :white]
+            [:green :black :red :red :red  :black :red :blue]
+            [:white :blue :white   :white  :green :green :red   :blue]
+            [:red :red   :white  :blue :green :green   :blue  :blue]
+            [:red   :blue  :green :blue :red   :white  :green :white]
+            [:red  :red :red :green   :red  :red :white   :blue]
+            [:blue :white :blue   :blue  :blue :blue :blue   :red]])
+
+(def fruit (map (fn [[x y color]]
+    (let [sprite (rs/sprite {:src "/img/fruit.png"
+                             :x (* x 64) :y (* y 64)
+                              :width 384
+                              :height 128
+                              :frame-width 32
+                              :frame-height 32
+                              :scale-x 2
+                              :scale-y 2})]
+      (rs/add-anim! sprite :red [0 11])
+      (rs/add-anim! sprite :green [12 23])
+      (rs/add-anim! sprite :blue [24 35])
+      (rs/add-anim! sprite :white [36 47])
+      (rs/play! sprite color true)
+      sprite)
+                    
+                    ) [ [1 0 :blue] 
```

Comment:
No idea why there's a big whitespace gap here.  But it's not making the code easier to read or understand.